internal/local/structure_v2.go

```diff
@@ -34,6 +34,7 @@ type AppStructureV2 struct {
 	Sync                  SyncStructure                     `json:"sync,omitempty"`
 	Secrets               SecretsStructure                  `json:"secrets,omitempty"`
 	LogForwarders         []map[string]interface{}          `json:"log_forwarders,omitempty"`
+	DataAPIConfig         map[string]interface{}            `json:"data_api_config,omitempty"`
```

Comment:
[q] as a more general question, when do we need to create a struct and when can we just use `map[string]interface{}`?